### DISCUSSION TITLE
📸 Introducing Publii Gallery Plugins!

### DISCUSSION THREAD

**Discussion by wolfcatworkshop** [GetPublii/Publii#1874] [Bug Report]
Question: 
- I updated the Mercury theme to 3.1.0.0
- Then I installed the Simple Lightbox Gallery
- I Synced the website 
Now whenever I click on a gallery image both gallery options show up, first the new one, then 
you hit ESC and then I have to hit ESC again because the older gallery preview is in effect as well.

How should I proceed to fix this issue? Thanks for your help!

**Comment by bobmitro** [comment]
> * Now whenever I click on a gallery image both gallery options show up, first the new one

It's not possible. You likely override the theme files `head.hbs` or/and `footer.hbs`, or you have a caching issue.

**Reply by wolfcatworkshop** [reply]
Oh boy, I missed one step after updating 🤦
Had to go to Site Settings → Basic Settings → Current theme: Mercury (v3.3.0.0)
So I had the old theme with the new plugin there for a minute. 

I assumed it was updated because when I checked on the three dot menu → Themes I 
could only see the updated theme. Oh well.

Thanks for the quick response! It got me looking in the right places.
